Test/Mono.Cecil.Tests/PortablePdbTests.cs

```diff
@@ -1148,5 +1148,33 @@ static void GetCodeViewPdbId (ModuleDefinition module, out byte[] pdbId)
 			buffer.WriteInt32 (cv.Directory.TimeDateStamp);
 			pdbId = buffer.buffer;
 		}
+
+		[Test]
+		public void WritePortablePdbPath ()
+		{
+			const string resource = "PdbPathLib.dll";
+			string destination = Path.GetTempFileName ();
+
+			using (var module = GetResourceModule (resource, new ReaderParameters { ReadSymbols = true })) {
+				module.Write (destination, new WriterParameters { WriteSymbols = true });
+			}
+
+			using (var module = ModuleDefinition.ReadModule (destination, new ReaderParameters { ReadSymbols = true })) {
+				GetCodeViewPdbPath (module, out string pdbPath);
+
+				Assert.AreEqual ("/_/artifacts/obj/PdbPathLib/release/PdbPathLib.pdb", pdbPath);
+			}
+		}
+
+		static void GetCodeViewPdbPath (ModuleDefinition module, out string pdbPath)
```

Comment:
nit: Is there a reason this uses an out parameter rather than returning a string?